src/docs/release/breaking-changes/1-22-deprecations.md

```diff
@@ -0,0 +1,768 @@
+---
+title: Deprecated API removed after v1.22
```

Comment:
Looking at the generated TOC, I don't think we need so many levels of headers. Maybe just h1 and h2. The deeper level of headers can be converted to bolded text or similar.

Reply:
I reduced the number of headings to h1,2,3. The third heading is for the individual class/change, which makes each of the changes in these guides linkable and accessible.